setup.cfg

```diff
@@ -26,6 +26,7 @@ mite_stats =
     mite = mite.stats:_MITE_STATS
     mite_http = mite_http:_MITE_STATS
     mite_selenium = mite_selenium:_MITE_STATS
+    mite_dns = mite_dns:_MITE_STATS
```

Comment:
I think it makes sense not to add this by default, to keep the default stats as lean as possible.  Other packages (like id-mite-nft) can add this line if they need dns timings.  (I would also not create the new mite_dns module, which seems like it should allow us to operate with the dns protocol directly.  I'd put the code in mite_http so that we have `mite_http:_DNS_STATS`)

Reply:
just to clarify: are you suggesting removal of the mite_dns/init.py file and moving everything over to mite_http? or just removal of this from setup?